### PR TITLE
BCDA-2339 Feature: Updating user guide to reflect varying sized creds for sandbox usage

### PR DESCRIPTION
**Pull request by tbellj**
Fixes [BCDA-2339](https://jira.cms.gov/browse/BCDA-2339)

Problem:
Before this change there was no way for our users to test different sized payloads in our synthetic environment. This change gives them a chance to emulate a more realistic scenario by more closely matching the size of the data they'll see in production.

Proposed Changes:
Added 4 different sized synthetic credentials for use in our sandbox environment.

Change Details:
_layouts/default.html - added script path to the accordion javascript.
assets/css/static-main.css - styling for the accordion.
assets/js/accordion.js - javascript for the accordion
sandbox/user_guide.md - added different sized ACO's for synthetic use.

Security Implications:
This change does **not** deal with any PII/PHI. The credentials added are completely synthetic and are only used for testing against our synthetic environment.

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

Acceptance Validation
Yes, I've locally tested this and it matches the design mockup. (Have yet to test each individual credential against our environment..stay tuned)

image

-------------------------------------------------------------------------------------------------

image

Feedback Requested:
Any and all

### CODE REVIEW

File: sandbox/user_guide.md [hunk 1]
```diff
@@ -59,6 +63,58 @@ f9780d323588f1cdfc3e63e95a8cbdcdd47602ff48a537b51dc5d7834bf466416a716bd4508e904a
 {%- endcapture -%}
 
 {% include copy_snippet.md code=client_secret %}
+</div>
+
+<button class="accordion"> Small ACO (2,500 synthetic beneficiaries) </button>
+<div class="acc_content">
+Client ID:
+{%- capture client_id -%}
+467bb940-7a40-4201-8aee-53d6015362fe
```
Comment (msnook) [inline sandbox/user_guide.md]: Were you able to get a token with all these values?

**Comment by msnook** [comment]
Looks nice!  
As for testing the creds - I suspect they may be incorrect.  Let me know how it goes.  I think you have the ones from pre-SSAS integration.

**Comment by msnook** [comment]
This is what I have:
```
OPENSBX

ACO Dev
3841c594-a8c0-41e5-98cc-38bb45360d3c
f9780d323588f1cdfc3e63e95a8cbdcdd47602ff48a537b51dc5d7834bf466416a716bd4508e904a

ACO Small
d5f83f74-6c55-4f1e-9d16-0022688171ba
01c23cf3f31f82a5c2dcef5136a8eaa32959158351f4e28aaec5fc6550cb2a5b5d112c5b8aacc434

ACO Medium 
8c75a6f6-02b9-4a47-96c1-0bd6efd4b5e3
e1c920141c4aca6b8f726fa8aa0f7b55e095fd1ea3368a5f24b3636fdc907f113d6677977c7259dd

ACO Large
f268a8c6-8a29-4d2b-8b92-263dc775750d
78ed083ad8e49475f36d04153649012b92c363a538ac97beaf0f7900403989581fc35324baa31e36

ACO Extra Large
6152afb4-c555-46e4-93de-fa16a441d643
1ace5a0e9fdf0c3d713e8c029269eacd024732ccd59e2cf283374e0f8dc93b34429d71b77e55b9a2
```

**Comment by epfrankle** [comment]
Confirmed, @msnook - I get a 401 Unauthorized error with the credentials in the PR, but am able to get tokens with the credentials in your comment.

File: sandbox/user_guide.md (hunk 1 above)
Comment (tbellj) [reply sandbox/user_guide.md]: changed to the newer values from this pr and tested all of them. Token responses for each one, nice!

**Comment by epfrankle** [comment]
Checking in one more time - @msnook @tbellj would you mind confirming that we've matched the right credentials with the right number of benes? (If the credentials we list under "large" do match up with 20k synthetic benes, would we be able to add one more set so we get the extra-large on there too?)

**Comment by msnook** [comment]
> Checking in one more time - @msnook @tbellj would you mind confirming that we've matched the right credentials with the right number of benes? (If the credentials we list under "large" do match up with 20k synthetic benes, would we be able to add one more set so we get the extra-large on there too?)

Yes, we should add the Extra Large test ACO as well.  @tbellj can you add that, I think it was part of the original ticket?
Also, @epfrankle I have created this ticket for static site release next Tues: https://jira.cms.gov/browse/BCDA-2439

**Comment by tbellj** [comment]
> > Checking in one more time - @msnook @tbellj would you mind confirming that we've matched the right credentials with the right number of benes? (If the credentials we list under "large" do match up with 20k synthetic benes, would we be able to add one more set so we get the extra-large on there too?)
> 
> Yes, we should add the Extra Large test ACO as well. @tbellj can you add that, I think it was part of the original ticket?
> Also, @epfrankle I have created this ticket for static site release next Tues: https://jira.cms.gov/browse/BCDA-2439

yeah i think it was in the original ticket, but was left out of the design mockup. Adding now...stay tuned

**Pull request merged**